### ISSUE TITLE
Paginator controls do not allow or have aria labels

### ISSUE THREAD

**Issue by alankstewart** [primefaces/primeng#11903]
### Describe the bug

While running accessibility tools like Accessibility Insights for Web, it is reported that the paginator's controls are missing aria-labels

### Environment

Windows 10, Microsoft Edge 104.0.1293.70

### Angular version

14.2.0

### PrimeNG version

14.0.2

### Build / Runtime

Angular CLI App

### Language

ALL

### Node version (for AoT issues node --version)

16.17.0

### Browser(s)

Microsoft Edge 104.0.1293.70

### Steps to reproduce the behavior

Run  a   Accessibility Insights for Web FastPass on any page with a p-table and paginator

### Expected behavior

Ability to specify aria-labels on paginator controls

**Comment by alankstewart** [comment]
Was this resolved in 15.0?

**Comment by alankstewart** [comment]
Possible duplicate of https://github.com/primefaces/primeng/issues/6812
